src/Execution/DataLoader/RelationBatchLoader.php

```diff
@@ -55,12 +56,12 @@ public function resolve(): array
 
         if ($this->paginationArgs !== null) {
             $modelRelationFetcher = new ModelRelationFetcher(
-                $this->getParentModels(),
+                $this->getParentModels(false),
```

Comment:
Without reading further, I have no idea what `false` does. Maybe use a separate function instead.

Reply:
Should be clearer now.